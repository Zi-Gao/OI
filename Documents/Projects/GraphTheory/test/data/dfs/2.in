252 789
229 186 31047
163 160 24241
69 60 22001
140 178 2079
115 100 22186
248 233 23031
36 144 15404
203 29 18294
7 124 25742
73 132 8313
204 60 14651
88 43 23517
68 193 29263
220 250 31384
19 165 19495
181 4 7180
176 89 14107
86 46 3183
38 118 29321
222 55 31853
51 251 15297
221 216 15691
214 178 3702
159 59 24370
196 175 14266
30 225 30571
219 86 32175
162 25 12980
180 37 11640
7 178 15353
177 137 19977
208 36 31246
22 158 20973
173 233 18247
182 137 15130
179 159 7162
182 114 2910
214 240 16717
108 241 24868
214 180 5098
42 214 4970
62 239 13922
205 113 20805
200 175 24938
182 75 2133
21 246 21562
72 121 29407
103 101 20916
166 212 31129
176 84 10016
174 132 24133
241 127 32660
8 104 15278
34 200 21364
128 124 7336
54 150 9598
125 19 3864
176 18 1894
107 86 31040
125 2 7661
98 169 26290
213 211 8435
141 2 20413
34 64 11774
246 193 22841
64 69 17440
109 192 1008
24 108 30284
61 76 2587
241 226 23926
207 126 3765
217 92 9742
131 136 5505
110 215 19645
210 10 19491
139 241 19566
137 241 19354
181 126 13308
125 109 30454
179 94 25642
238 157 18217
189 245 9903
157 11 32059
119 2 30065
251 230 11089
81 65 13642
2 133 6717
71 143 27127
96 224 5278
118 97 4334
54 234 25008
220 60 28505
79 221 2391
125 114 17258
203 93 10268
243 59 952
226 148 29955
47 243 14196
110 87 29835
92 57 2192
213 208 31545
170 66 3368
244 131 26485
90 170 30681
116 96 11692
226 85 30401
247 63 17131
189 145 5984
119 152 29045
8 48 6550
8 27 2
236 225 18022
86 115 8099
222 238 12118
149 140 7554
211 251 16761
55 224 11520
180 33 25479
17 35 5071
169 120 30866
226 1 25892
248 57 10556
195 16 6679
9 207 21360
224 149 1215
100 234 1301
230 13 10429
171 146 51
140 108 16163
195 213 1916
221 4 25860
87 22 10300
120 251 31504
74 212 29469
164 86 25086
121 147 10051
25 46 21187
115 157 19366
7 73 25271
221 113 8377
116 117 9024
246 70 18623
12 49 2461
96 227 3935
120 200 14046
234 168 26956
113 82 8499
121 222 11047
145 86 22292
105 172 8930
222 159 27126
73 102 16714
5 135 20866
227 165 27014
77 238 6542
38 74 23209
136 38 15426
238 178 2988
234 227 30520
40 1 19560
16 131 1392
100 251 7790
194 28 16754
238 141 3784
191 242 6647
42 100 30220
121 181 18664
251 84 30277
150 119 12189
121 17 26276
37 183 27806
32 44 10992
13 164 32337
36 163 18737
97 41 25439
131 221 31246
72 3 29875
156 81 201
65 105 32356
129 229 24048
108 116 13284
34 12 31762
173 56 20808
169 27 22549
113 104 24208
76 105 28421
193 219 27435
158 50 31101
2 248 1848
115 216 16615
101 59 3365
92 106 13647
68 204 18883
170 17 27268
110 59 2541
38 196 22958
248 75 23623
91 131 16556
54 217 15050
168 94 19159
244 31 8980
174 61 26355
71 53 13753
173 206 23678
251 203 5121
91 194 26275
72 36 12510
90 183 25962
189 183 29947
167 143 9733
123 245 16175
130 1 26959
90 216 16842
76 210 15313
165 164 21087
222 73 29931
126 140 17701
183 205 23282
134 235 11785
200 34 2147
129 38 12536
214 130 21118
86 194 14365
88 154 11231
21 152 16884
81 207 24052
213 179 21134
98 231 2659
234 192 14936
55 51 16823
136 17 10812
106 120 12016
20 93 16382
222 187 28619
121 48 10014
117 101 16347
21 214 27048
51 223 12494
140 148 27060
107 141 21612
38 117 1932
194 187 3384
72 101 428
58 145 16570
131 86 29468
16 106 29398
24 29 18898
45 82 29637
35 128 29210
13 72 21123
21 214 5437
171 21 17364
12 59 30940
57 164 12841
205 178 6583
57 145 17681
178 162 22491
11 206 28408
74 8 17260
224 135 1634
170 161 8322
38 74 11236
185 79 24022
44 16 20617
45 87 19764
54 100 10340
53 50 15847
179 130 17278
102 35 30878
99 40 31592
127 102 22904
110 176 28171
153 3 12350
139 128 28314
22 207 30706
96 247 15273
212 196 11827
72 32 30169
175 38 26682
171 62 21658
230 202 1607
174 123 32498
191 152 26593
54 40 19528
19 88 4666
239 161 5536
206 80 10123
52 128 14812
212 75 10576
223 131 6086
210 100 6303
202 82 5920
112 213 28444
118 34 30187
28 84 28721
116 244 409
121 199 17806
92 36 15350
174 169 2860
11 200 14816
137 23 11834
207 180 24106
231 11 8131
160 220 32064
77 12 9963
123 224 1635
139 134 26648
106 40 5525
183 8 19815
1 192 23954
185 95 12846
87 10 15204
35 34 22148
153 51 3446
142 226 268
60 128 9781
98 86 9323
137 64 30364
147 142 11058
29 145 26424
236 119 2090
224 181 22168
208 49 8768
242 4 27160
233 209 31360
120 93 23172
203 140 20452
229 176 14361
123 1 22500
231 12 22969
91 145 13316
104 205 22904
190 81 27606
168 246 13516
182 73 12332
87 122 28931
154 15 22377
183 201 5324
26 247 11321
36 245 28150
201 33 7952
128 226 1721
135 192 70
116 205 28696
16 188 27560
27 196 12103
231 152 20438
185 196 14870
200 122 19143
66 30 4819
19 154 16963
194 101 19999
238 197 31299
22 197 11685
214 14 17510
32 43 4288
32 144 2820
207 147 22631
192 9 16263
56 153 3584
165 246 26381
227 76 13541
104 148 29063
238 182 15551
128 107 1736
63 193 7943
195 232 21067
243 173 23627
239 201 24977
48 152 24075
141 113 12906
199 248 18499
229 162 25990
44 12 4614
86 207 32538
11 21 24240
87 125 653
233 64 6556
86 189 22360
40 135 21601
199 172 5285
130 201 27646
126 242 24150
16 57 4999
53 166 14705
240 52 8997
59 198 30505
212 35 144
160 80 23809
197 168 12876
188 47 21939
43 41 16711
97 19 2530
222 82 1203
88 50 4301
219 42 25132
76 16 18537
156 215 145
220 67 8632
109 99 30225
124 230 6459
200 92 10841
82 96 31941
59 149 3061
234 193 22304
70 31 25649
136 241 12099
48 118 5924
9 35 3081
204 219 1455
56 25 11945
191 237 9615
245 119 1374
249 180 18390
112 26 17668
155 186 104
207 107 13197
76 176 16720
18 232 18082
5 3 4865
42 28 8399
227 2 28396
4 101 31998
19 201 13286
209 251 2876
192 26 1113
225 243 29423
115 68 11320
103 136 28177
136 77 291
127 111 3596
77 95 6476
3 103 1599
229 90 17903
161 70 8546
195 195 1155
216 170 7735
132 84 9714
114 141 25944
131 115 19820
147 222 11509
235 90 12471
177 54 225
148 186 16919
47 230 5195
246 173 12305
22 108 4068
234 15 443
234 53 2670
150 250 5725
82 38 21224
69 28 1419
23 91 31429
111 211 22781
127 156 29699
219 127 27604
161 67 27120
113 74 29457
18 208 3331
137 188 24923
31 59 10519
237 59 3190
247 162 23262
196 22 17653
93 249 25717
111 244 259
156 145 19031
21 249 24934
229 182 22820
158 236 17410
225 118 18929
49 76 22910
48 25 15913
119 191 23532
177 46 3353
16 130 8818
70 152 11458
37 233 29289
135 79 11406
60 149 10360
246 239 5102
21 128 13446
154 223 20499
194 37 31141
94 91 829
181 62 32568
137 8 19799
146 59 4874
78 187 22745
95 148 5137
168 57 26660
30 87 11276
103 12 7893
9 128 21057
192 110 8134
5 66 7171
35 94 9140
158 160 12239
123 53 13758
157 76 13846
108 62 6716
70 126 28439
177 103 22951
25 1 14165
50 128 19581
123 39 19590
222 115 9832
99 134 8983
83 79 11803
33 12 8340
148 33 23706
75 106 10646
227 236 23011
139 219 12803
215 63 1595
210 103 29896
187 124 18061
222 203 13398
237 114 28564
188 150 30591
186 141 16078
198 249 11923
198 107 8551
155 164 6851
230 132 26785
157 98 17979
85 114 18525
208 106 25967
133 107 26582
67 251 10889
196 239 13742
170 61 31242
221 77 26586
244 106 7147
80 165 25873
249 59 1767
152 71 6344
9 184 19389
32 29 13286
38 251 1151
138 88 27568
121 16 4172
197 35 13729
213 124 24077
213 68 6694
62 73 10506
50 161 20857
36 173 24683
238 215 17678
201 216 3853
164 154 5457
28 107 2860
194 185 13990
1 134 3533
160 213 27110
187 236 8186
101 46 5076
200 22 8222
145 170 5178
136 70 3120
64 231 2225
121 231 19819
84 181 12352
193 71 2234
215 101 15445
96 153 820
162 93 4295
119 157 3117
228 189 6115
154 117 9929
157 99 12459
177 37 27426
178 135 11479
204 71 23368
247 12 8506
93 214 17111
137 99 18063
211 25 14292
161 244 8676
182 61 2567
105 247 29464
123 195 27159
140 27 19751
208 238 25608
139 166 3898
175 119 249
211 51 1720
138 129 4646
190 67 8646
64 54 23457
125 22 5877
166 242 7999
107 247 4638
239 75 30892
53 41 8074
109 147 8608
155 163 1540
225 65 8620
136 65 21858
229 230 12062
137 178 29350
40 198 12003
221 225 11216
61 176 24439
203 224 11498
3 16 24384
200 163 23806
214 17 30953
8 143 1270
226 75 153
212 156 31637
243 95 8979
176 10 14595
150 34 25642
145 162 31149
58 16 15372
155 217 19298
20 135 23374
59 62 20726
223 130 11202
23 139 1998
17 170 14460
150 1 14375
172 183 20884
140 138 17396
198 37 4227
195 25 4518
116 150 26673
138 64 10048
16 133 14345
111 206 32633
130 178 31666
178 184 1513
15 232 14657
38 50 18540
74 108 19958
132 103 6975
32 129 15852
16 248 31364
208 177 17726
161 213 12083
136 5 10641
82 96 30370
130 247 16648
144 67 29478
173 29 24576
29 181 20859
186 246 28976
116 112 8694
201 82 29700
240 28 13751
106 83 2884
112 63 12102
92 108 14951
216 234 13159
45 30 31270
1 90 4621
139 178 32042
40 12 10412
87 251 25576
42 127 30305
178 243 19205
70 48 12196
118 66 8675
161 57 25220
59 166 7964
206 198 15768
75 31 1816
1 177 6959
234 174 23812
116 48 24420
115 188 4300
78 50 21277
79 14 23797
207 243 16166
91 83 23902
208 135 29147
73 160 1103
43 231 24361
77 29 1387
245 49 32206
102 230 9811
90 251 13843
205 154 13092
69 212 30084
1 158 10522
76 158 6221
85 140 5102
244 124 9279
175 86 10968
175 67 6835
10 215 58
50 53 15807
205 107 9047
26 214 23164
120 177 30119
82 235 19790
240 114 15789
192 195 22123
100 18 16213
41 122 17425
115 151 26176
88 211 17718
108 101 26763
180 99 12941
168 198 18683
97 66 5430
102 210 21900
235 39 31096
77 6 15313
54 42 336
127 188 7861
85 181 32193
111 15 20850
29 95 26402
199 206 2523
129 201 11233
140 121 4173
110 97 19058
251 3 15843
43 103 22115
77 247 2501
148 178 12755
97 70 3234
214 228 15281
137 62 19894
67 184 18584
224 93 28406
176 78 16161
120 87 30250
222 223 29181
24 9 3876
151 133 8511
204 137 18264
234 236 14739
17 180 32116
98 113 17243
148 205 8157
178 54 19692
203 98 27403
148 176 16982
60 25 13629
54 234 27271
113 67 1800
61 132 29864
220 44 22683
137 107 29181
19 35 23009
191 167 6069
214 122 16312
201 237 365
105 244 15102
38 63 14417
43 87 3625
36 130 19492
213 100 15510
53 145 8849
193 157 6876
201 75 5776
171 227 10697
26 92 25811
128 150 11725
147 53 870
197 232 29438
121 36 12815
92 194 12804
136 102 7698
191 102 9343
91 76 12127
103 214 6097
90 34 20063
131 208 17234
15 160 30924
220 107 25419
156 146 8550
80 7 11353
168 93 26971
61 204 8966
99 125 5051
50 111 20909
52 16 29272
202 137 19219
131 121 18646
121 8 29238
192 45 3553
179 39 12348
159 21 23487
224 99 10126
69 67 6325
