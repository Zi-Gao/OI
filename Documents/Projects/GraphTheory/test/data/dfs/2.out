1 90 34 12 49 76 16 57 145 86 46 115 68 193 71 53 41 122 50 111 15 160 80 7 73 102 35 94 91 83 79 14 221 4 101 59 62 239 75 31 106 40 135 192 9 128 107 141 2 133 248 233 64 54 42 28 84 181 126 140 27 196 22 108 116 48 25 118 66 30 87 10 215 63 125 19 88 43 103 136 5 3 17 170 61 132 176 18 208 36 130 178 162 93 214 180 33 37 183 8 104 148 186 246 70 152 173 29 95 56 153 51 223 131 121 147 142 226 85 114 222 55 224 99 134 235 39 149 82 38 74 212 156 81 65 105 172 244 124 230 13 72 32 44 129 201 216 234 168 198 249 174 123 195 213 100 251 203 98 113 67 184 169 120 177 137 23 139 166 242 219 127 188 47 243 150 119 157 11 21 200 92 194 185 187 236 225 163 175 206 191 167 143 237 250 241 231 179 159 211 232 245 227 165 164 154 117 229 182 144 202 247 207 146 96 238 197 199 205 228 189 240 52 78 89 204 60 161 77 6 109 97 112 26 158 138 217 69 209 45 110 210 220 151 